engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -3189,3 +3215,42 @@ enum ViewFocusDirection {
   /// This is typically result of the user pressing shift + tab.
   backward,
 }
+
+/// A request to evaluate the content of a view at a specific position.
+///
+/// See also:
+///
+/// * [PlatformDispatcher.onHitTest], the callback that the platform
+///   invokes to hit test a view at a specific position.
+/// * [HitTestResponse], the result of a hit test request.
+class HitTestRequest {
+  /// Creates a hit test request.
+  const HitTestRequest({required this.view, required this.offset});
```

Comment:
consider make this a private constructor and move the HitTestRequest wrapping to _hitTest method in this file.

Reply:
what is "move the HitTestRequest wrapping"?

Reply:
I still need to construct the request in unit test tho